Automattic/wp-calypso
# E2E: add new `atomicUser` for use in Atomic E2E testing.

#### Proposed Changes

This PR adds a new user with Business plan `atomicUser` to take the place of `eCommerceUser`.

#### Testing Instructions

Ensure the following build configurations are passing:

Additionally verification if required:

1. pull the branch.
2. decrypt the secrets file with `yarn workspace wp-e2e-tests decrypt-secrets`.
3. verify there is entery for `atomicUser`.


#### Pre-merge Checklist





Related to #
